be/CMakeLists.txt

```diff
@@ -532,6 +543,10 @@ set(STARROCKS_LINK_LIBS
     ${WL_END_GROUP}
 )
 
+if (${WITH_DLA_CACHE} STREQUAL "ON")
+    set(STARROCKS_LINK_LIBS ${STARROCKS_LINK_LIBS} Cache)
```

Comment:
if this is dla cache, I suggest we use lib and directory name like `dla_cache`.

Reply:
> 

OK, as the cache module implement a common block cache, so I think `block_cache` is more suitable.